src/components/dialogs/AddEvidenceDialog.vue

```diff
@@ -400,7 +401,31 @@ export default {
       hotkeys: "config/hotkeys",
       config: "config/fullConfig",
       credentials: "auth/credentials",
+      configTags: "config/tags",
+      sessionItems: "sessionItems",
     }),
+    filteredTags() {
```

Comment:
Please calculate this each time the dialog is displayed and then save it so we don't recompute it many times.